tests/integration/test_ethereum_tester.py

```diff
@@ -319,9 +310,6 @@ class TestEthereumTesterEthModule(EthModuleTest):
     test_eth_fee_history_no_reward_percentiles = not_implemented(
         EthModuleTest.test_eth_fee_history_no_reward_percentiles, MethodUnavailable
     )
-    test_eth_send_transaction_with_nonce = not_implemented(
```

Comment:
Is this the right approach for these? The tests are passing now but I'd like to know if py-evm was updated or if something else changed?